src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -187,7 +185,7 @@ def get_rate(cls, svc_type, svc_name, path):
         data = mgr.get_counter(svc_type, svc_name, path)[path]
 
         if data and len(data) > 1:
-            return differentiate(*data[-2:])
+            return derivative(*data[-2:])
```

Comment:
That syntax is totally ok, but might require some extra attention (and if someone tries with 3 samples `derivative(*data[-3:])`, it'd still work... but weirdly). So, what about?:
```suggestion
            return derivative(data[-2], data[-1])
```

Reply:
This seems like an improvement for a different PR to me?

Reply:
It's just a change in the way the function is fed. I don't think it qualifies as an improvement.

Reply:
As the API of this function stays unchanged, I'd argue for keeping the noise in this PR small.

Reply:
The splat operator is a language feature for feeding functions with a variable number of arguments (or when there're plenty of them). As in this case we're already checking that the list will have at least 2 elements, I don't understand why we should do any trick.

Reply:
We're dealing with the tail of a list aka `data[-2:]`, meaning it will obviously not properly work, if you call it with `reverse(data[-2:])`. We're not dealing with two individual parameters. I'm :+1: for keeping it this way.

Reply:
I will change the condition above to expect 2 elements in the list.